module.nix

```diff
@@ -38,6 +38,11 @@ in {
                     default = "/sys/devices/pci0000:00/0000:00:18.3/hwmon/hwmon2/temp1_input";
                     description = "path to the cpu thermal sensor, probably something in /sys/device";
                 };
+                GPUThermalZone = mkOption {
+                    type = types.nullOr types.str;
+                    default = "";
```

Comment:
Missing default. Please use the old value("device/hwmon/hwmon1/temp1_input") for backwards compatibility.